### PR TITLE
libmypaint: update to 1.5.0

### PR DESCRIPTION
**Pull request by teh-jazzman**
Taking over for previous maintainer with permission.

Update is needed for subsequent update of mypaint package.

gimp and opentoonz link to the current library and will probably need to be updated as well. Is there some sort of proper process for this?

### CODE REVIEW

**Comment by kartikynwa** [comment]
Not sure if it's needed but I'm okay with the change in maintainership.

**Comment by teh-jazzman** [comment]
Hm, not sure what to do about the failing architectures. Should I just disable then? Or does anyone know how to get them to compile?

File: srcpkgs/libmypaint/template [hunk 1]
```diff
@@ -23,6 +23,9 @@ libmypaint-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		if test -e "${DESTDIR}/usr/share/gir-1.0"; then
```
Comment (xtraeme) [inline srcpkgs/libmypaint/template]: use `if [ "$build_option_gir" ]; then` instead

File: common/shlibs [hunk 2]
```diff
@@ -592,7 +592,7 @@ libecpg_compat.so.3 postgresql-libs-8.4.2_1
 libecpg.so.6 postgresql-libs-8.4.2_1
 libpgtypes.so.3 postgresql-libs-8.4.2_1
 libpq.so.5 postgresql-libs-8.4.2_1
-libmypaint-1.3.so.0 libmypaint-1.3.0_1
+libmypaint-1.5.so.1 libmypaint-1.5.0_1
```
Comment (xtraeme) [inline common/shlibs]: probably revdeps are broken due to this, see `xbps-query -RX libmypaint`

Comment (teh-jazzman) [reply common/shlibs]: So I should keep the old entry and just add the new one?

File: srcpkgs/libmypaint/template (hunk 1 above)
Comment (teh-jazzman) [reply srcpkgs/libmypaint/template]: The changed if condition apparently is true even when the gir-1.0 directory doesn't exist, which makes the vmove fail on several architectures.

File: common/shlibs (hunk 2 above)
Comment (kartikynwa) [reply common/shlibs]: No. I think you'll need to bump revision numbers of packages that you see in `xbps-query -RX libmypaint`

File: srcpkgs/libmypaint/template (hunk 1 above)
Comment (teh-jazzman) [reply srcpkgs/libmypaint/template]: I removed the check altogether now.

File: common/shlibs (hunk 2 above)
Comment (teh-jazzman) [reply common/shlibs]: OK, added the revision bumps. Checks fail due to timeout but building works fine on my system.

**Comment by xtraeme** [comment]
1.5.1 is out

**Pull request closed without merging**